statement-summary-tables.md

```diff
@@ -89,26 +90,32 @@ The table schema of `statements_summary_history` is identical to that of `statem
 
 The fields `SUMMARY_BEGIN_TIME` and `SUMMARY_END_TIME` represent the start time and the end time of the historical time range.
 
-## `cluster_statements_summary` and `cluster_statements_summary_history`
+## `statements_summary_evicted`
 
-`statements_summary` and `statements_summary_history` display the statement summary data of only a single TiDB server. To query the data of the entire cluster, you need to query `cluster_statements_summary` and `cluster_statements_summary_history`.
+The `tidb_stmt_summary_max_stmt_count` variable controls the maximum number of statements that the `statement summary` table stores in memory. The `statement summary` table uses the LRU algorithm. Once the number of SQL statements exceed the `tidb_stmt_summary_max_stmt_count` value, the longest unused record is evicted from the table. The number of evicted SQL statements during each period are recorded in the `statements_summary_evicted` table.
```

Comment:
Should change "`statement summary` table" to "`statements_summary` table"
"`statement summary history` table" -> `statements_summary_history` table

Reply:
```suggestion
The `tidb_stmt_summary_max_stmt_count` variable controls the maximum number of statements that the statement summary tables store in memory. The statement summary tables use the LRU algorithm. Once the number of SQL statements exceeds the `tidb_stmt_summary_max_stmt_count` value, the longest unused record is evicted from the tables. The number of evicted SQL statements during each period is recorded in the `statements_summary_evicted` table.
```